src/officehours_api/consumers.py

```diff
@@ -124,11 +124,15 @@ def trigger_queue_update(sender, instance: Queue, created, **kwargs):
     if instance.deleted:
         return
     transaction.on_commit(lambda: send_queue_update(instance.id))
+    for host in instance.hosts.all():
+        transaction.on_commit(lambda: send_user_update(host.id))
```

Comment:
@jlost, I just added this. Based on @teshsopher 's feedback, I realized that we probably should subscribe to queue name and status changes.